src/main/java/org/logstash/filters/geoip/Databases.java

```diff
@@ -0,0 +1,126 @@
+                    Fields.COUNTRY_CODE2,
+                    Fields.COUNTRY_NAME,
+                    Fields.CONTINENT_NAME
+            )
+    ),
+    DOMAIN(
+            "GeoIP2-Domain",
+            EnumSet.of(
+                    Fields.DOMAIN
+            )
+    ),
+    ASN(
+            "GeoLite2-ASN",
+            EnumSet.of(
+                    Fields.IP,
+                    Fields.AUTONOMOUS_SYSTEM_NUMBER,
+                    Fields.AUTONOMOUS_SYSTEM_ORGANIZATION
+            )
+    ),
+    ISP(
+            "GeoIP2-ISP",
+            EnumSet.of(
+                    Fields.IP,
+                    Fields.AUTONOMOUS_SYSTEM_NUMBER,
+                    Fields.AUTONOMOUS_SYSTEM_ORGANIZATION,
+                    Fields.ISP,
+                    Fields.ORGANIZATION
+            )
+    ),
+    ANONYMOUS_IP(
+            "GeoIP2-Anonymous-IP",
+            EnumSet.of(
+                    Fields.HOSTING_PROVIDER,
+                    Fields.TOR_EXIT_NODE,
+                    Fields.ANONYMOUS_VPN,
+                    Fields.ANONYMOUS,
+                    Fields.PUBLIC_PROXY,
+                    Fields.RESIDENTIAL_PROXY
+            )
+    ),
+    ENTERPRISE(
+            "Enterprise",
+            EnumSet.of(
+                    Fields.IP,
+                    Fields.COUNTRY_CODE2,
+                    Fields.COUNTRY_NAME,
+                    Fields.CONTINENT_NAME,
+                    Fields.REGION_ISO_CODE,
+                    Fields.REGION_NAME,
+                    Fields.CITY_NAME,
+                    Fields.LOCATION
+            )
+    ),
+    UNKNOWN(
+            "Unknown",
+            EnumSet.noneOf(Fields.class)
+    );
+
+    private final String databaseType;
+    private final Set<Fields> defaultFields;
+
+    Databases(String databaseType, final Set<Fields> defaultFields) {
+        this.databaseType = databaseType;
+        this.defaultFields = defaultFields;
+    }
+
+    public Set<Fields> getDefaultFields() {
+        return Collections.unmodifiableSet(defaultFields);
+    }
+
+    public static Databases fromDatabaseType(final String type) {
+        // It follows the same com.maxmind.geoip2.DatabaseReader#getDatabaseType logic
+        if (type.contains(CITY.databaseType)) {
+            return Databases.CITY;
+        } else if (type.contains(COUNTRY.databaseType)) {
+            return Databases.COUNTRY;
+        } else if (type.contains(DOMAIN.databaseType)) {
+            return Databases.DOMAIN;
+        } else if (type.contains(ASN.databaseType)) {
+            return Databases.ASN;
+        } else if (type.contains(ISP.databaseType)) {
+            return Databases.ISP;
+        } else if (type.contains(ENTERPRISE.databaseType)) {
+            return Databases.ENTERPRISE;
+        } else if (type.contains(ANONYMOUS_IP.databaseType)) {
+            return Databases.ANONYMOUS_IP;
+        }
+
+        // Maybe, we should throw and exception here instead of use an unknown database type.
```

Comment:
Any thoughts on this?

Reply:
If the pipeline have multiple geoip-filters and one of it is UNKNOWN, throwing exception in register phase will cause pipeline restart. I prefer to keep backward compatible and add a log to say the current database type and give warning if it is UNKNOWN

Reply:
Done ✅

Reply:
Have you added a log to user to warn them the database type is unknown?

Reply:
Done 👍